slp2mp4/slp-to-mp4.py

```diff
@@ -193,4 +215,6 @@ def main():
 
 if __name__ == '__main__':
     installDependencies()
+    patch_dolphin_sys_game_settings()
```

Comment:
This is a style/preference thing, but I think main() should be the only thing here; these other functions should just be be called from within main itself.

My thinking is that the whole `if __name__ == '__main__':` thing is just boilerplate that has to be there; I prefer not to put any actual logic there other than the call to main

Reply:
Makes sense